build.sh

```diff
@@ -264,8 +264,8 @@ do
         GROUP_ID=50
       fi
       {
-        cat share/docker/Dockerfile
-        grep -vF 'FROM avro-build-ci' share/docker/DockerfileLocal
+        sed -r "s/openjdk:8/openjdk:${JAVA:-8}/g" share/docker/Dockerfile
+        [ ${JAVA:-8} -le 8 ] && grep -vF 'FROM avro-build-ci' share/docker/DockerfileLocal
```

Comment:
Maybe we should use multi-stage builds with targets instead of modifying the dockerfile in place. Or it could be something to do in the future.

Reply:
I think that would definitely be an improvement, but probably in the context of https://issues.apache.org/jira/browse/AVRO-2630 (which targets an overall improvement of Travis CI with Yetus).

For the moment, I'd just like to see a JDK11 local build that's as easy as the JDK8 local build.